XPlat.Devices.Launcher/Launcher.Android.cs

```diff
@@ -26,12 +28,8 @@ public Launcher()
         /// </param>
         public Launcher(Context context)
         {
-            this.Context = context;
         }
 
-        /// <summary>Gets or sets the Android context to be used for launching an Activity with.</summary>
```

Comment:
With the changes you've reverted, you will need to add the context back into this and update references throughout the class with the original implementation.